x-pack/test/login_selector_api_integration/apis/login_selector.ts

```diff
@@ -56,6 +56,7 @@ export default function ({ getService }: FtrProviderContext) {
       'authentication_realm',
       'lookup_realm',
       'authentication_provider',
+      'authentication_type',
     ]);
 
     expect(apiResponse.body.username).to.be(username);
```

Comment:
Same as the previous comment (though this test does already check for `authentication_provider`)!